src/components/map/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useRef, useEffect } from 'react';
+import mapboxgl from 'mapbox-gl';
+import style from './map.module.css';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import { mapboxData } from '@atb/modules/org-data';
+
+export type Position = {
+  lat: number;
+  lng: number;
+};
+
+export type MapProps = {
+  initialPosition?: Position;
```

Comment:
Why is this `initialPosition` and not just `position`? `initial` implies to me that it is only set on initialized component, but the semantics of the component here listens for change on position. So it's not just initial position but just "position"
```suggestion
  position?: Position;
```

Reply:
You are correct. I initially thought the component would update the position, but I can see that listening for position change is probably better.